src/store/reducers/packagesReducer.js

```diff
@@ -107,6 +131,7 @@ export const redux = createReducerScenario({
   insertPackage: { type: 'PACKAGE_INSERT', reducer: insertPackageReducer },
   deletePackage: { type: 'PACKAGES_DELETE', reducer: deletePackageReducer },
   updatePackages: { type: 'PACKAGES_UPDATE', reducer: updatePackagesReducer },
+  updateConfigTypes: { type: 'CONFIG_TYPES_UPDATE', reducer: updateConfigTypesReducer },
```

Comment:
Should this have "PACKAGES_" in the type name since its part of the packages reducer?

Reply:
done